modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/CppRestClientCodegen.java

```diff
@@ -110,6 +112,10 @@ public CppRestClientCodegen() {
         addOption(DEFAULT_INCLUDE,
                 "The default include statement that should be placed in all headers for including things like the declspec (convention: #include \"Commons.h\" ",
                 this.defaultInclude);
+        addOption(GENERATE_INTERFACES_FOR_APIS,
+                "Generate abstract base classes (interfaces) for APIs. This allows to mocks APIs (e.g. for unit testing).");
```

Comment:
Should be `This allows to **mock** APIs`

Reply:
Yep, will fix it.